src/main/java/org/fundacionjala/sfdc/util/CommonActions.java

```diff
@@ -212,6 +212,7 @@ public static void autoCompleterLightTheme(final String textToSelect) {
      */
     public static boolean istWebElementPresentOnList(final List<WebElement> listOfElements,
                                                      final String element) {
+        System.out.println(element);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)